TERMS.md

```diff
@@ -0,0 +1,121 @@
+## Service
```

Comment:
I suggest that we add a few words on the intention of this document. Something like below to make it clear that this describe what we want, not how it currently is.

*This document describes the intended meaning and usage of the elements defined in the VSC syntax. It reflects the wanted target state of VSC and it currently does not correspond 100% to what has been implemented in VSC and VSC-tools!*